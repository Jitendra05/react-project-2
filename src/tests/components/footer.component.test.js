import React from 'react';
import Footer from '../../components/footer.component';
import {shallow} from 'enzyme';

test('should render Footer component correctly',()=>{
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});