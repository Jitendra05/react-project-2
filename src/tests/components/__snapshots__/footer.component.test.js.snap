// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer component correctly 1`] = `
<div
  className="footer"
>
  <div
    className="container"
  >
    <h4
      className="header__subtitle"
    >
      Udemy React Course Project @ Part 2
    </h4>
  </div>
</div>
`;
